stylesheets/app.scss

```diff
@@ -201,10 +202,9 @@ button, .button {
     margin-left: 20px;
   }
   &.blue {
-    background:$med-blue;
-    &:hover {
-      background: lighten($med-blue,10%);
-    }
+    background:white;
```

Comment:
Yes, this would affect all buttons with the CSS class 'blue'. But there's actually no current buttons with that class on site (only on jboss.org pages that were leftover but no longer used). 

UX removed the button this change was made for on the last moment of the review process. But I still left the changes for future buttons that have this look, which is the right look for 'blue' buttons according to VD.